applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -43,17 +44,37 @@ static bool delete_broadcast_src;
 
 uint8_t audio_mapping_mask[CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT] = {UINT8_MAX};
 
+/* Advertising data for peer connection */
 #if (CONFIG_AURACAST)
+/* Location of the first byte for the PBA features and parental rating */
+#define PBA_FEATURE_START		(BT_UUID_SIZE_16)
+#define BROADCAST_PARENTAL_RATING_START (BT_UUID_SIZE_16 + 4)
+
 /* Make sure pba_buf is large enough for a 16bit UUID and meta data
  * (any addition to pba_buf requires an increase of this value)
  */
-NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 8);
-static struct bt_data ext_ad[5];
-#else
-static struct bt_data ext_ad[4];
+static uint8_t pba_buf[] = {BT_BYTES_LIST_LE16(0x1856),
```

Comment:
Quite a few magic values here. Needs a comment or defines.

Reply:
Change this one to BT_UUID_PBA_VAL, will also look at the rest.